setup.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
+
+# This script sets up a Ubuntu machine to run this Node.js app locally
+
+
+# Check if node is in apt-cache
+if [ $(apt-cache show nodejs) ]
```

Comment:
For if conditionals in bash you can just use parenthesis for command exit code like this:

```
if ( apt-cache show nodejs )
...
```

The brackets are for implicitly running the `test` command.

Reply:
We are no longer checking the apt-cache for `nodejs`

Reply:
I have changed the tests to use parenthesis